src/components/Assemblies/CabinetRole.svelte

```diff
@@ -1,21 +1,67 @@
 <script lang="ts">
+	import type { AssemblyMember } from '$lib/datasheets/assembly-member';
 	import type { CabinetSeat } from './shared';
 
 	export let cabinets: CabinetSeat[];
 	export let role: CabinetSeat['role'];
+
+	const show = (event: MouseEvent, member: AssemblyMember) => {
+		const memberName = `${member.firstname} ${member.lastname}`;
+		const tooltipElem = document.getElementById('tooltip');
+
+		if (tooltipElem) {
+			tooltipElem.setAttribute(
+				'style',
+				`left: ${event.layerX}px; top: ${event.layerY + 20}px; display: flex;`
+			);
+		}
+
+		const memberElem = document.getElementById('member-name');
+		const roleElem = document.getElementById('role');
+
+		if (memberElem) {
+			memberElem.textContent = memberName;
+		}
+
+		if (roleElem) {
+			roleElem.textContent = member.assemblyRole.role;
+		}
+	};
+
+	const hide = () => {
+		const tooltipElem = document.getElementById('tooltip');
+
+		if (tooltipElem) {
+			tooltipElem.setAttribute('style', `display: none;`);
+		}
+	};
 </script>
 
 <div class="role">
 	<p class="heading-compact-01">{role}</p>
 	<div class="group-dot">
 		{#each cabinets.find((c) => role === c.role)?.parties || [] as party}
-			{#each Array.from({ length: party.count || 0 }, (_, i) => i) as _}
-				<span class="dot" style="background-color: {party.color || '#8D8D8D'};"></span>
+			{#each party?.members ?? [] as member}
+				<!-- svelte-ignore a11y-no-static-element-interactions -->
+				<span
+					class="dot"
+					style="background-color: {party.color || '#8D8D8D'};"
+					on:mouseenter={(e) => show(e, member)}
+					on:mouseleave={hide}
```

Comment:
the function name should be `showTooltip` and `hideTooltip`, without tooltip people might not know what will it show/hide.

Reply:
Done